test/cli.js

```diff
@@ -364,6 +364,14 @@ test('bails when config contains `"tap": true` and `"watch": true`', t => {
 	});
 });
 
+test('bails when config does not contain `--concurrency` and `-c` values', t => {
```

Comment:
@novemberborn This test fails with error below:

```
bails when config does not contain `--concurrency` and `-c` values
  not ok spawn /Users/yatharthk/.node-spawn-wrap-62339-36f9a5f0c8ca/node ENOENT
    at:
      line: 1026
      column: 11
      file: util.js
      function: exports._errnoException
    code: ENOENT
    errno: ENOENT
    syscall: spawn /Users/yatharthk/.node-spawn-wrap-62339-36f9a5f0c8ca/node
    path: /Users/yatharthk/.node-spawn-wrap-62339-36f9a5f0c8ca/node
    spawnargs:
      - ../../../cli.js
      - test.js
    test: bails when config does not contain `--concurrency` and `-c` values
```

I'm not really sure what's expected here. Can you also let me know if `{dirname: 'fixture/concurrency'}` is correct? I didn't understand how this works!